cl/opinion_page/urls.py

```diff
@@ -34,6 +35,11 @@
         name="court_publish_page",
     ),
     # Opinion pages
+    path(
+        "opinion/<int:pk>/<blank-slug:slug>/visualizations/",
+        visualizations_deprecation_redirect,
+        name="cluster_visualizations",
+    ),
```

Comment:
Check out cl/urls.py, where we usually put our redirections. I think you can use the approach there to avoid having a view too.

Reply:
Thanks, good suggestion, I'm so used to doing it the Django way that I didn't think about this 😅